theme/templates/resource-landing-page/top-right-buttons.html

```diff
@@ -87,6 +87,18 @@
                         {% endif %}
                     {% endif %}
                 {% endif %}
+
+                <a id="btn-download-all" 
+                    title="Download all content as Zipped BagIt Archive"
+                    {% if cm.raccess.require_download_agreement %}
+                    data-target="#license-agree-dialog-bag" data-toggle="modal"
+                    data-placement="auto"
+                    {% endif %}
+                    data-bag-url="{{ cm.bag_url }}">
+                    <span data-toggle="tooltip" data-placement="bottom"
+                            class="fa fa-cloud-download icon-button icon-blue"></span>
+                </a>
```

Comment:
```suggestion
                <a id="btn-download-all" 
                    {% if cm.raccess.require_download_agreement %}
                    data-target="#license-agree-dialog-bag" data-toggle="modal"
                    data-placement="auto"
                    {% endif %}
                    data-bag-url="{{ cm.bag_url }}">
                    <span data-toggle="tooltip" data-placement="bottom"
                            title="Download all content as Zipped BagIt Archive"
                            class="fa fa-cloud-download icon-button icon-blue"></span>
                </a>
```
☝️ the "title" needs to be on the `<span>` otherwise the tooltip will not show properly.